fcall.go

```diff
@@ -107,11 +107,6 @@ type Fcall struct {
 }
 
 func newFcall(tag Tag, msg Message) *Fcall {
-	switch msg.Type() {
```

Comment:
clientnegotiate and servernegotiate both hardcode NOTAG to the call. Negotiation to my rather pedantic server also works with 9pr.